src/routes/Wallets/routes/Create/WalletsCreateViewContainer.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+import { connect } from 'react-redux'
+
+import {
+  openView,
+  closeView,
+  goToNextStep,
+  goToPrevStep,
+  changeNameInput,
+  changePasswordInput,
+  changePasswordHintInput,
+  changePasswordConfirmInput,
+} from './modules/walletsCreate'
+
+import WalletsCreateView from './WalletsCreateView'
+
+const mapStateToProps = ({
+  wallets,
+  walletsCreate,
+}: State): {|
+  +invalidFields: FormFields,
+  +name: string,
+  +password: string,
+  +passwordHint: string,
+  +passwordConfirm: string,
+  +currentStep: Index,
+  +isLoading: boolean,
+  +isPasswordExists: boolean,
+|} => ({
+  ...walletsCreate,
+  isPasswordExists: !!wallets.testPasswordData,
+})
+
+const mapDispatchToProps: {|
+  +openView: Function,
```

Comment:
Here we don't need to add type for mapDispatchToProps. It will be done by flow automatically.
Edge of that `Function `can broke quality of type check here

Reply:
https://github.com/jibrelnetwork/jwallet-web/commit/475b6c6f29a48da4eee70f1eb6827761ba1c7e2e#diff-35e90e0cb63d12bfdb148d49e0771d08R34